test/test_normal_form_game.jl

```diff
@@ -260,6 +284,69 @@
                 end
             end
         end
+
+        @testset "Test is_dominated" begin
+            coordination_game_matrix = [4 0; 3 2]
+            player = Player(coordination_game_matrix)
+            for action = 1:num_actions(player)
+                @test !is_dominated(player, action)
+            end
+
+            payoffs_2opponents = Array{Int64}(undef, 2, 2, 2)
```

Comment:
```suggestion
            payoffs_2opponents = Array{Int}(undef, 2, 2, 2)
```

Reply:
There are multiple existing tests that use `Int64` -- should this change be made there as well?